.github/workflows/performance.yml

```diff
@@ -160,16 +170,27 @@ jobs:
           npm run env:cli -- import themeunittestdata.wordpress.xml --authors=create --path=/var/www/${{ env.LOCAL_DIR }}
           rm themeunittestdata.wordpress.xml
 
+      - name: Deactivate WordPress Importer plugin
+        run: npm run env:cli -- plugin deactivate wordpress-importer --path=/var/www/${{ env.LOCAL_DIR }}
+
       - name: Update permalink structure
-        run: |
-          npm run env:cli -- rewrite structure '/%year%/%monthnum%/%postname%/' --path=/var/www/${{ env.LOCAL_DIR }}
+        run: npm run env:cli -- rewrite structure '/%year%/%monthnum%/%postname%/' --path=/var/www/${{ env.LOCAL_DIR }}
 
       - name: Install additional languages
         run: |
           npm run env:cli -- language core install de_DE --path=/var/www/${{ env.LOCAL_DIR }}
           npm run env:cli -- language plugin install de_DE --all --path=/var/www/${{ env.LOCAL_DIR }}
           npm run env:cli -- language theme install de_DE --all --path=/var/www/${{ env.LOCAL_DIR }}
```

Comment:
Is there a way we can install languages only for required themes (Twenty Twenty-One, Twenty Twenty-Three, Twenty Twenty-Four)? Currently, it installs languages for all themes (Twenty Eleven, Twenty Fifteen, etc.). Alternatively, can we remove those themes from the WP build?

Reply:
Hmm we probably could install only those select translations, but I don‘t see a need for it. It would be just yet another list we‘d have to maintain, which is easy to forget. Also the installing is quick anyway.